# message for storing info from darpa status replies
float32 time_remaining
string score
uint8 remaining_reports 